core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -219,7 +219,7 @@ public static function substrHtml($strString, $intNumberOfChars)
 	 *
 	 * @return string The decoded string
 	 */
-	public static function decodeEntities($strString, $strQuoteStyle=ENT_QUOTES, $strCharset=null)
+	public static function decodeEntities($strString, $strQuoteStyle=ENT_QUOTES|ENT_HTML5, $strCharset=null)
```

Comment:
I think we should keep the default here and use `$strQuoteStyle|ENT_HTML5` below.

Reply:
Then we could not pass an argument without `| ENT_HTML5` anymore.

Reply:
That should be fine I think?

Reply:
I don’t see a use case for using `html_entity_decode()` without `ENT_HTML5` (encode can be different, but not decode IMO).

Reply:
Maybe there is one, or maybe not. But if we hardcode `| ENT_HTML5`, people have no option to override anymore. Therefore, I would rather only adjust the default value.

Reply:
I’d still prefer adding `| ENT_SUBSTITUTE | ENT_HTML5` below and only use the `$strQuoteStyle` parameter for selecting the desired quote-style (ENT_COMPAT, ENT_QUOTES, ENT_NOQUOTES).
But I’m fine either way.

Reply:
Changed in 0cd7f956136e216be7db5e306145a86141f25756.